go/tests/endpoint/endpoint.go

```diff
@@ -54,6 +54,20 @@ type fn func()
 
 var cleanup fn = func() {}
 
+var cwd, err_cwd = os.Getwd()
```

Comment:
change the way you name these errors to be more golang-idiomatic: something like cwdErr / binErr